File: ignored-test/java-junit/ClassOne_TestOneOneIgnoredWithCause.java
import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.Disabled;

import static org.junit.jupiter.api.Assertions.assertTrue;

public class ClassOne {

    @Test
    @Disabled("This is the ignore cause")
    public void testOneOneIgnoredWithCause() {
        assertTrue(1 == 2, "This is ignored with cause, no failure");
    }
}

File: ignored-test/CSharpXUnit/CSharpXUnit.Tests/Tests/ClassOne_TestOneOneIgnoredWithCause.cs
using Xunit;

namespace TestAssetsXunit
{
    public class ClassOne
    {
        [Fact(Skip = "This is the ignore cause")]
        public void TestOneOneIgnoredWithCause()
        {
            {
                Assert.True(1 == 2, "This is ignored with cause, no failure");
            }
        }
    }
}
